src/GenAPI/Microsoft.DotNet.GenAPI.Task/GenAPITask.cs

```diff
@@ -49,6 +49,11 @@ public class GenAPITask : TaskBase
         /// </summary>
         public bool IncludeVisibleOutsideOfAssembly { get; set; }
 
+        /// <summary>
+        /// Includes assembly attributes which are values that provide information about an assembly.
+        /// </summary>
+        public bool IncludeAssemblyAttributes { get; }
```

Comment:
Please also expose the property in the .targets file. All task properties should be exposed with msbuild properties in the targets file.

Reply:
Updated